src/ethereum/frontier/vm/instructions/system.py

```diff
@@ -0,0 +1,110 @@
+"""
+Ethereum Virtual Machine (EVM) System Instructions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Implementations of the EVM system related instructions.
+"""
+from ethereum.base_types import U256, Uint
+from ethereum.utils.address import compute_contract_address
+
+from .. import Evm, Message
+from ..gas import (
+    GAS_CREATE,
+    GAS_ZERO,
+    calculate_gas_extend_memory,
+    subtract_gas,
+)
+from ..memory import extend_memory, memory_read_bytes
+from ..stack import pop, push
+
+
+def create(evm: Evm) -> None:
+    """
+    Creates a new account with associated code.
+
+    Parameters
+    ----------
+    evm :
+        The current EVM frame.
+    """
+    # This import causes a circular import error
+    # if it's not moved inside this method
+    from ethereum.frontier.vm.interpreter import (
+        STACK_DEPTH_LIMIT,
+        process_create_message,
+    )
+
+    endowment = pop(evm.stack)
+    memory_start_position = Uint(pop(evm.stack))
+    memory_size = pop(evm.stack)
+
+    gas_cost = GAS_CREATE + calculate_gas_extend_memory(
+        evm.memory, memory_start_position, memory_size
+    )
+    evm.gas_left = subtract_gas(evm.gas_left, gas_cost)
+    extend_memory(evm.memory, memory_start_position, memory_size)
+    sender_address = evm.env.origin
+    sender = evm.env.state[sender_address]
+    if (sender.balance < endowment) or (
+        evm.message.depth + 1 > STACK_DEPTH_LIMIT
+    ):
+        push(evm.stack, U256(0))
+        return None
+
+    call_data = memory_read_bytes(
+        evm.memory, memory_start_position, memory_size
+    )
+
+    evm.env.state[evm.message.storage_address].nonce += 1
+    contract_address = compute_contract_address(
+        evm.message.storage_address,
+        evm.env.state[evm.message.storage_address].nonce - 1,
+    )
+
+    gas_left = evm.gas_left
+    evm.gas_left = subtract_gas(evm.gas_left, gas_left)
+
+    child_message = Message(
+        caller=evm.message.storage_address,
+        current=b"",
+        gas=gas_left,
+        value=endowment,
+        data=b"",
+        code=call_data,
+        create_address=contract_address,
+        storage_address=contract_address,
+        depth=evm.message.depth + 1,
+    )
+    child_evm = process_create_message(child_message, evm.env)
+    push(evm.stack, U256.from_be_bytes(child_evm.message.storage_address))
+    evm.gas_left = child_evm.gas_left
+
+
+def return_opcode(evm: Evm) -> None:
```

Comment:
Yuck. Stupid keyword clashes.

According to [PEP 8](https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments), we can use `return_` which might be more consistent with the other opcode implementations?